src/inmanta/ast/__init__.py

```diff
@@ -65,6 +65,9 @@ def merge(self, other: "Location") -> "Location":
 
         return Location(self.file, min(self.lnr, other.lnr))
 
+    def as_anchor_target(self, docstring: Optional[str]) -> "AnchorTarget":
+        return AnchorTarget(location=self, docstring=docstring)
```

Comment:
I think we have two options here:
- `AnchorTarget(Location)` and we use this method to create it
- `AnchorTarget` has a location field, no need for this method.